scripts/processing/05-run_ica.py

```diff
@@ -30,19 +31,28 @@ def run_ica(subject_id, tsss=None):
         raws.append(mne.io.read_raw_fif(run_fname))
     raw = mne.concatenate_raws(raws)
     if tsss:
+        # SSS reduces the data rank and the noise levels, so let's include more
+        # ICA components here
```

Comment:
does it reduce the number of components or is that you need a higher fraction to get the same number of components as in non-SSS data?

Reply:
SSS reduces rank, to recover the right number of components using ICA we need it to be this high

Reply:
yes, but I guess what I was saying is that it's not clear what you mean by "more". I guess you meant more compared to what it would do for n_components = 0.98. But it could also mean: more number of components than non-SSS data